test/WebSites/ConnegWebSite/Controllers/FallbackOnTypeBasedMatchController.cs

```diff
@@ -40,7 +40,7 @@ public IActionResult FallbackGivesNoMatch(int input)
         public IActionResult OverrideTheFallback_UsingCustomFormatters(int input)
         {
             var objectResult = new ObjectResult(input);
-            objectResult.Formatters.Add(new HttpNotAcceptableOutputFormatter());
+            objectResult.ExcludeFormatterMatchOnObjectTypeOnly = true;
```

Comment:
Do we need to add a test which covers the exact scenario which was reported ? I do not see any functional test changes,